docs/getting-started/datapackage-find-prepare-share-guide.md

```diff
@@ -4,59 +4,56 @@ date: 2018-04-26
 authors: Dmytro Gierman, Branko Djordjevic
 ---
 
-**In this guide we describe the way to gather the information and to make it presentable and easy to use for your customers.**
+**In this guide, we describe a way to gather information and make it presentable and easy to use for your customers.**
+
+Over time we've seen an increase in the adoption of the internet, with so much information available to us it can quickly become overwhelming. With this enormous data, numerous opportunities exist for those that can make sense of these datasets and present them in a clear and succinct manner.
 
-With the proliferation of internet one cannot escape the overwhelming information all around us. With this much data, fortunes are made by those that can make sense of datasets and present it in a clear and succinct manner.
 
 [[toc]]
 
 # What is Data Package
 
 Let's say that you obtained data about pharmaceutical drug spending by countries and that you already have it saved in a [comma separated values](https://en.wikipedia.org/wiki/Comma-separated_values) (CSV) file.
 
-You wish to present the data to the appropriate audience and csv format is great for tabular representation. The user can clearly see what ammount in which year has the country spent on pharmaceuticals. But there are things that csv file can't show us. We don't know for example if in the column **TIME** there are only integer type years like 1972 or will we find something like April 1972 which is string.
+You wish to present the data to the appropriate audience and CSV format is great for tabular representation. The user can see what amount in which year has the country spent on pharmaceuticals. But there are things that CSV file can't show us. We don't know for example if in the column **TIME** there are only integer type years like 1972 or will we find something like April 1972 which is string?
 
 Users also may want to see:
 - general description of the data
 - information about the sources 
 - license information
-- data file structure information
-- etc
+- data file structure information etc.
 
 All this information is called **metadata** and should be stored alongside with the data to make the information frame around the raw data you have, so users could understand the context.
 
-And here comes the idea of [**Data Package**](https://datahub.io/docs/data-packages).
-
-**Data Package** - *is a data plus a meta-information about it.*
+[**Data Package**](https://datahub.io/docs/data-packages) - *is a data plus a meta-information about it.*
```

Comment:
```
And this is where [*Data Package**] comes in. Data Package is ... 
```

Reply:
Note also that i prefer out of band links for markdown as they are cleaner e.g.

```
[Data Package][dp] is ...

[dp]: https://dathaub.io/docs/data-packages
```

Rather than ... 

```
[Data Package](https://datahub.io/docs/data-packages)
```